executor/src/test/java/org/k8loud/executor/kubernetes/KubernetesBaseTest.java

```diff
@@ -27,7 +32,11 @@ public void setUp() {
         client = server.getClient();
 
         lenient().when(kubernetesClientProviderMock.getClient()).thenReturn(client);
-        kubernetesService = new KubernetesServiceImpl(kubernetesClientProviderMock);
+        lenient().doAnswer(i -> DataStorageTestUtil.store(i.getArgument(0), i.getArgument(1)))
```

Comment:
can't you use these lenient methods where needed? we don't wont to add it everywhere because it creates more confusion

Reply:
I can, just lazy; refactored